scripts/utils/env-variable-value.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+echo_env_variable_value() {
+  local -n REF=$1
```

Comment:
On macOS the default bash version installed is 3.2 which does not support the `-n` option to the `local` command.

Consider using this instead: `local REF=$(printenv $1)`